src/components/UI/Overlay/OverlayContent/ConfirmationContext.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { FunctionComponent } from "react";
+import { useOverlayContext } from "../../../../common/context/OverlayContext";
+import { Button } from "../../Button";
+import { IconSuccess } from "../../Icon";
+import warningIcon from "../../../../../src/static/images/Alert/warning.png";
+import { useLockBodyScroll } from "../../../../common/hooks/useLockBodyScroll";
+
+export const enum MESSAGE_TYPE {
+  SUCCESS = "success",
+  WARNING = "warning",
+  NONE = "none",
+}
+
+export interface ConfirmationContextProps {
+  messageType: MESSAGE_TYPE;
+  title?: string;
+  children?: React.ReactNode;
+  btnText?: string;
+  maxHeight?: number;
+  className?: string;
+}
+
+export const DismissalButton: FunctionComponent<{ buttonText?: string }> = ({
+  buttonText,
+}: {
+  buttonText?: string;
+}) => {
+  const { closeOverlay } = useOverlayContext();
+  return (
+    <Button
+      className="bg-cerulean-500 hover:bg-cerulean-800 rounded-xl px-3 py-2 mx-auto mt-3 text-white font-normal"
+      onClick={closeOverlay}
+    >
+      {buttonText ? buttonText : "Dismiss"}
+    </Button>
+  );
+};
+
+export const ConfirmationContext: FunctionComponent<
+  ConfirmationContextProps
```

Comment:
the naming of `ConfirmationContext`, particularly the suffix of `Context`, might be easily confused with react's useContext naming convention